File: spring/actuator/src/main/java/com/hyfly/actuator/entity/validate/Validate.java
package com.hyfly.actuator.entity.validate;

import com.alibaba.fastjson2.annotation.JSONField;
import lombok.Data;

import java.util.List;

@Data
public class Validate {

    @JSONField(name = "valid")
    private Boolean valid;

    @JSONField(name = "error_count")
    private Integer errorCount;

    @JSONField(name = "warning_count")
    private Integer warningCount;

    @JSONField(name = "diagnostics")
    private List<Diagnostic> diagnostics;
}

File: terraform/src/main/kotlin/com/hyfly/tf/actuator/entity/validate/Validate.kt
package com.hyfly.tf.actuator.entity.validate

import com.alibaba.fastjson2.annotation.JSONField

class Validate {
    @JSONField(name = "valid")
    val valid: Boolean? = null

    @JSONField(name = "error_count")
    val errorCount: Int? = null

    @JSONField(name = "warning_count")
    val warningCount: Int? = null

    @JSONField(name = "diagnostics")
    val diagnostics: MutableList<Diagnostic>? = null
}
